src/CryptoInterface.h

```diff
@@ -133,6 +141,14 @@ void BSL_CryptoInit(void);
  */
 void BSL_CryptoDeinit(void);
 
+//#if defined(BSL_CRYPTO_RNG_FN_OVERRIDE)
```

Comment:
Since this is implemented as a callback, don't worry about the preprocessor conditional. Just add a statement above the parameter doc: `@warning Intended to be used only for testing. Providing an alternative RNG may break FIPS-140 compatibility.`